docs/ReferenceManyInput.md

```diff
@@ -0,0 +1,287 @@
+| `helperText`   | Optional | `string`                  | -                                | Text to be displayed under the input                                                                                                                                |
+| `source`       | Optional | `string`                  | `id`                             | Name of the field that carries the identity of the current record, used as origin for the relationship                                                              |
+| `filter`       | Optional | `Object`                  | -                                | Filters to use when fetching the related records, passed to `getManyReference()`                                                                                    |
+| `perPage`      | Optional | `number`                  | 25                               | Maximum number of referenced records to fetch                                                                                                                       |
+| `sort`         | Optional | `{ field, order }`        | `{ field: 'id', order: 'DESC' }` | Sort order to use when fetching the related records, passed to `getManyReference()`                                                                                 |
+| `defaultValue` | Optional | `array`                   | -                                | Default value of the input.                                                                                                                                         |
+| `validate`     | Optional | `Function` &#124; `array` | -                                | Validation rules for the array. See the [Validation Documentation](./Validation.md) for details.                                   |
+| `sx`           | Optional | `SxProps`                 | -                                | MUI shortcut for defining custom styles                                                                                                                             |
+
+## `children`
+
+`<ReferenceManyInput>` creates an `<ArrayInputContext>`, so it accepts the same type of children as `<ArrayInput>`: a Form iterator. React-admin bundles one such iterator: `<SimpleFormIterator>`. It renders one row for each related record, giving the user the ability to add, remove, or edit related records.
+
+```jsx
+<ReferenceManyInput reference="variants" target="product_id">
+    <SimpleFormIterator>
+        <TextInput source="sku" />
+        <SelectInput source="size" choices={sizes} />
+        <SelectInput source="color" choices={colors} />
+        <NumberInput source="stock" defaultValue={0} />
+    </SimpleFormIterator>
+</ReferenceManyInput>
+```
+
+Check out [the `<SimpleFormIterator>` documentation](./SimpleFormIterator.md) for more details.
```

Comment:
There is no SimpleFormIterator.md file.
Could be a reference to the ArrayInput one? I'm not clear on that.

Reply:
That's totally right, and it's missing! I'll add one section about SimpleFormIterator in this PR.